### DISCUSSION TITLE
Font size in tables

### DISCUSSION THREAD

**Discussion by apcamargo** [pradyunsg/furo#558] [Questions] [answered]
Hi,

I'm writing a documentation with lots of numerical tables that feel way too big. I want to reduce the font table size, but I couldn't find the CSS variable. Is there a specific variable that controls this?

**Comment by pradyunsg** [comment]
Do you have any example page for me to look at for this?

**Reply by apcamargo** [reply]
I don't have anything online right now (only [this](https://portal.nersc.gov/genomad/score_calibration.html) page), but here's an indication of the area I want to reduce the size of the fonts:

![Untitled 3](https://user-images.githubusercontent.com/22940964/196787033-ac6bd809-a2c0-418a-913f-32f882ceed45.png)

This specific table looks ok, but I'd prefer smaller fonts because the larger tables are looking too conspicuous.

**Comment by pradyunsg** [comment, marked as answer]
To answer the question asked: No, there's no direct CSS variable for this.

You can inject CSS, as documented in https://pradyunsg.me/furo/customisation/injecting/, to achieve this:

```css
table {
   font-size: 0.875rem;  /* using rem to respect end user's browser configuration of font sizes */
}
```

**Comment by apcamargo** [comment]
That's perfect! Thanks!
